spec/mongoid/traversable_spec.rb

```diff
@@ -251,4 +251,87 @@
       end
     end
   end
+
+  describe "#discriminator_key" do
+
+    context "when the discriminator key is not changed" do 
+      it "equals _type" do
+        expect(Instrument.discriminator_key).to eq("_type")
+      end
+
+      it "children's discriminator keys equal _type" do
+        expect(Guitar.discriminator_key).to eq("dkey")
+        expect(Piano.discriminator_key).to eq("_type")
+      end
+
+      it "the base discriminator key is _type" do
+        expect(Mongoid.discriminator_key).to eq("_type")
+      end
+    end
+
+    context "when the discriminator key is changed at the base level" do 
+      before do
+        Mongoid.discriminator_key = "hello"
+      end
+
+      after do
+        Mongoid.discriminator_key = "_type"
```

Comment:
Can this be assigned `nil` such that subsequent code would use the global default?

Reply:
did you mean to put this comment on Instrument.discriminator_key? 

Also, setting the `Instrument.discriminator_key` to nil changes the global default because we delegate the `discriminator_key` method to `::Mongoid`

Reply:
Yes I meant this comment to be on a class.